docker/host.sls

```diff
@@ -53,5 +53,13 @@ docker-py:
     - reload_modules: True
 {% endif %}
 
+{%- if host.registry is defined %}
+
+docker_login:
+  cmd.run:
```

Comment:
Dockerio is deprecated since version 2015.8.0. Dockerng does not seem to have docker login module.